job-server/src/main/resources/application.conf

```diff
@@ -17,6 +17,12 @@ spark {
       rootdir = /tmp/spark-jobserver/filedao/data
     }
 
+    # To load up job jars on startup, place them here,
+    # with the app name as the key and the path to the jar as the value
+    # job-jar-uris {
+    #   test = ../job-server-tests/target/scala-2.10/job-server-tests_2.10-0.5.3-SNAPSHOT.jar
```

Comment:
This currently only supports local paths. Adding support for other initial jar locations could be supported, though...is there a distinct need for that currently, or should we start with just local JARs?